src/differential_arithmetic.jl

```diff
@@ -16,7 +16,7 @@ Notice:
 
 Base.:+(::DoesNotExist, ::DoesNotExist) = DoesNotExist()
 Base.:*(::DoesNotExist, ::DoesNotExist) = DoesNotExist()
-for T in (:One, :AbstractThunk, :Any)
+for T in (:One, :AbstractThunk, :Composite, :Any)
```

Comment:
Does this even make sense?

Reply:
Why wouldn't it?

Reply:
Oh yes, sorry this was ment to be on `One`